opt/build/onnxruntime/pack.sh

```diff
@@ -2,6 +2,7 @@
 
 set -e
 VER="$1"
+PLATFORM="$2"
```

Comment:
`PLATFORM=$($READIES/bin/platform -t)` should be used here.
And `linux-${PLATFORM}` should be replaced with `${PLATFORM}`.